packages/core/src/providers/linkedin.ts

```diff
@@ -84,33 +72,20 @@ export default function LinkedIn<P extends LinkedInProfile>(
     id: "linkedin",
     name: "LinkedIn",
     type: "oauth",
```

Comment:
you can actually remove most of the configurations:
- type should be `oidc`
- `authorization` can be removed 
- `profile` can be removed

You can take a look at the Okta provider for example.